modules/provision-oci-infrastructure-ocp-cluster.adoc

```diff
@@ -6,16 +6,60 @@
 [id="provision-oci-infrastructure-ocp-cluster_{context}"]
 = Provisioning {oci} infrastructure for your cluster
 
-By using the {ai-full} to create details for your {product-title} cluster, you can specify these details in a stack. A stack is an {oci} feature where you can automate the provisioning of all necessary {oci} infrastructure resources, such as the custom image, that are required for installing an {product-title} cluster on {oci}.
-
-The {oci-first} Compute Service creates a virtual machine (VM) instance on {oci}. This instance can then automatically attach to a virtual network interface controller (vNIC) in the virtual cloud network (VCN) subnet. On specifying the IP address of your {product-title} cluster in the custom manifest template files, the {oci} instance can communicate with your cluster over the VCN.
+When using the {ai-full} to create details for your {product-title} cluster, you now specify these details in a Terraform stack. A stack is an {oci} feature that automates the provisioning of all necessary {oci} infrastructure resources that are required for installing an {product-title} cluster on {oci}.
 
 .Prerequisites
 
-* You uploaded the discovery ISO image to the {oci} bucket. For more information, see "Using the {ai-full} to generate an {oci}-compatible discovery ISO image".
+* You downloaded the discovery ISO image to a local directory. For details, see "Using the {ai-full} to generate an {oci}-compatible discovery ISO image" in this chapter.
+* You downloaded the Terraform stack template to a local directory. For details, see "Preparing the OCI environment" in this chapter.
```

Comment:
Want to avoid self-reference, but also due to the modularity of content and the published output being subject to change, this could become an untrue statement
```suggestion
* You downloaded the discovery ISO image to a local directory. For details, see "Using the {ai-full} to generate an {oci}-compatible discovery ISO image".
* You downloaded the Terraform stack template to a local directory. For details, see "Preparing the OCI environment".
```